### PR TITLE
feat:main resourcers projects, fix: styles, #1286

### PR DESCRIPTION
**Pull request by Kamui-Hebertt**
rendering main resources links from teams on the page  #1286

### CODE REVIEW

**Comment by yacineKahlerras** [comment]
Hey hebert can you link this to a task from zenhub in the developement section ?

File: apps/site-projects/src/components/modules/Resources/ProjectResources/TeamLinks/ResourceCard.js [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
\ No newline at end of file
+
+import { ResourcersCards, ResourcersIcon, CardsBtn, ResourcersIconInfo } from "./StyledResourceCard";
+
+function ResourceCard({resource : {img, title, guideTitle,guideLink, moreInfo}}) {
+    return (
+           <ResourcersCards>
+            <ResourcersIcon src={`/images/projects/resources/logos/${img}`}  alt={title}/>
+            <a href={guideLink} target="_blank">
+            <CardsBtn>{title}</CardsBtn>
```
Comment (yacineKahlerras) [inline apps/site-projects/src/components/modules/Resources/ProjectResources/TeamLinks/ResourceCard.js:9]: Hey Hebertt, great work everything is in order, just one thing we need to use the UI elements from UI packages so we have a universal design on the website, so you can do : 

```
import Button from "@devlaunchers/components/src/components/atoms/Button";

<Button buttonSize="standard" buttonType="secondary">{title}</Button>
```
i merged `developement/new-builds` branch containing the updated theme into this branch so you can use those UI components.

File: apps/site-projects/src/components/modules/Resources/ProjectResources/TeamLinks/StyledResourceCard.js [hunk 2]
```diff
@@ -1,11 +1,68 @@
...
+    flex-direction: row;
+    gap:2rem;
+    background: white;
+    border: 3px solid #528cb0 ;
+    border-radius: 1rem;
+    padding: 2rem 3rem;
+    margin: 4rem 0;
+    padding-bottom: 2rem;
+    flex-wrap: wrap;
+    justify-content: center;
```
Comment (yacineKahlerras) [inline apps/site-projects/src/components/modules/Resources/ProjectResources/TeamLinks/StyledResourceCard.js:15]: also here to make the elements wrap just like the design and not centered with flex should maybe look something like this : 
```
export const ResourcesContainer = styled.fieldset`
    display:grid;
    grid-template-columns: repeat(auto-fit,minmax(15rem,16rem));
    justify-content: center;
    gap:2rem;
    background: white;
    border: 3px solid #528cb0 ;
    border-radius: 1rem;
    padding: 2rem 3rem;
    margin: auto 0;
    padding-bottom: 2rem;
`;
```
i made it a grid so the elements are centered but wrap elements just like the design, you tweak and play around with it until it feels right


68747470733a2f2f696d616765732e7a656e68756275736572636f6e74656e742e636f6d2f3634393231313832623861323337306464626630663962322f37376466613937352d646130662d346666392d393233642d353864616164343662656264

**Review by yacineKahlerras — CHANGES_REQUESTED** [review]

File: apps/site-projects/src/components/modules/Resources/ProjectResources/TeamLinks/StyledResourceCard.js (hunk 2 above)
Comment (Kamui-Hebertt) [reply apps/site-projects/src/components/modules/Resources/ProjectResources/TeamLinks/StyledResourceCard.js]: thank you, i've done that.

**Review by yacineKahlerras — APPROVED** [review]
everything is good now, thank you

**Pull request merged**